src/absil/illib.fs

```diff
@@ -36,13 +42,19 @@ let reportTime =
             let first = match !tFirst with None -> (tFirst := Some t; t) | Some t -> t
             printf "ilwrite: TIME %10.3f (total)   %10.3f (delta) - %s\n" (t - first) (t - prev) descr
             tPrev := Some t
+#endif
 
 //-------------------------------------------------------------------------
 // Library: projections
 //------------------------------------------------------------------------
 
-[<Struct>]
 /// An efficient lazy for inline storage in a class type. Results in fewer thunks.
+#if FABLE_COMPILER
+type InlineDelayInit<'T when 'T : not struct>(f: unit -> 'T) = 
```

Comment:
When you use ``#if FABLE_COMPILER`` please add a comment indicating what Fable doesn't support, or use a feature define like ``#if FSLANG_NO_MUTABLE_STRUCTS``.  The latter helps document that the F# language as supported by Fable has some specific semantic differences to F# as supported by .NET.